joomla/joomla-cms
# adds support for screen readers to active menu item

Pull Request for Issue # .

### Summary of Changes
adds `aria-current="page"` to active menu item.

### Testing Instructions
select a menu item to make it active.
view source for active menu item.
should look like "actual result" image.
apply patch
`aria-current="page"` should be before the `>` in the anchor for the "active" menu item.
as shown in the "expected result" image.

Now select or create as needed a alias menu item
repeat above steps.

### Expected result

![image](https://user-images.githubusercontent.com/1850089/64746860-d8fa1700-d4d2-11e9-8217-d8411b70df92.png)


### Actual result
![image](https://user-images.githubusercontent.com/1850089/64746971-41e18f00-d4d3-11e9-8fab-7ccdde1ef466.png)


### Documentation Changes Required

None
